providers/aws/components/es/setup.ftl

```diff
@@ -66,20 +103,6 @@
 
     [#local AccessPolicyStatements = [] ]
 
-    [#local esAccounts = getAWSAccountIds( solution.Accounts )]
```

Comment:
This code was to support access to shared ES instances. Are we removing it on the basis of using links for this type of situation in the future?

Reply:
Yeah I think we can look at it again once links are added. It also had some issues with the deny rules not working properly